os/se/ameba/Kconfig

```diff
@@ -52,7 +52,6 @@ config HW_SE_STORAGE
 
 config SE_SECURE_CONTEXT_SIZE
 	int "Secure Context Size"
-	default 6144
+	default 9216
```

Comment:
Then, how about making a dependency with the board?
```
default 6144 if AMEBAD_TRUSTZONE
default 9216 if AMEBALITE_TRUSTZONE
```
Because you use this config for both.

Reply:
Hi @sunghan-chang ,
We have made changes to it based on your suggestion.
Thank You.